### ISSUE TITLE
Adding some additional terms.

### ISSUE THREAD

**Issue by williamchern** [billwuqin/draft-bclp-green-terminology#27]
After double check on the following standards, 
- ITU-T L.1330
- ETSI ES 203 228 V1.4.1 (2022-04) 
- IEEE 802.3az-2003
I suggest to add the following definitions as mentioned in the above standards.
- power consumption; 
- energy consumption; 
- energy saving features; 
- energy-efficiency ethernet;
- site energy effciency;
If some of these terms are agreed to be added, I can provide a pull request for them, thanks.

**Comment by billwuqin** [comment]
Leave these terms aside, I review https://datatracker.ietf.org/doc/draft-pignataro-green-enviro-sust-terminology/, a set of terms are relevant as follows:
1.Energy, power, and their measurement
2.Energy efficiency (EE)
3.Energy proportionality 
4.Energy savings / conservation (ES) 
I think the first 2 terms have already been covered, for Energy proportionality, try to understand how it is different from energy efficiency, PUE, can we use it to measure the ratio between energy consmption by linecard and energy consumption of the whole network device?
For Energy saving or Energy Conservation, I think it is the effect of implementing the enegy efficiency strategy.

**Comment by billwuqin** [comment]
> After double check on the following standards,
> 
> * ITU-T L.1330
> * ETSI ES 203 228 V1.4.1 (2022-04)
> * IEEE 802.3az-2003
>   I suggest to add the following definitions as mentioned in the above standards.
> * power consumption;
> * energy consumption;
> * energy saving features;
> * energy-efficiency ethernet;
> * site energy effciency;
>   If some of these terms are agreed to be added, I can provide a pull request for them, thanks.

Are these terms core terms, my impression many of these terms are specific to a certain technologies or scenarios? 
Energy saving features is useful, but it seems defintions is not self contained, relies on some context information.

**Comment by williamchern** [comment]
I agree, Energy saving features relies on the context, and  EEE is certain technology, I thought that well-known technology for energy saving is good to be clarified here. 
Anyway, the term of "site energy efficiency" though coming from wireless related  ETSI standard, but it shares the same context for all network devices like optical and datacom devices, etc.

**Comment by billwuqin** [comment]
site energy efficiency is 3GPP specific term.
